### ISSUE TITLE
IndexError

### ISSUE THREAD

**Issue by stealth0414** [lucastabelini/PolyLaneNet#62]
Hi,
I am trying to test the trained model on my own dataset. I modified the config but I get the following error in models.py

lower_loss = mse(target_lowers[valid_lanes_idx], pred_lowers[valid_lanes_idx])
 IndexError:The shape of the mask [16, 1] at index 0 does not match the shape of the indexed tensor [80, 1] at index 0
  File "D:\py\智能车\2022drive\S弯检测\PolyLaneNet-master\lib\models.py", line 116, in loss
    lower_loss = mse(target_lowers[valid_lanes_idx], pred_lowers[valid_lanes_idx])
  File "D:\py\智能车\2022drive\S弯检测\PolyLaneNet-master\train.py", line 54, in train
    loss, loss_dict_i = criterion(outputs, labels, **criterion_parameters)
  File "D:\py\智能车\2022drive\S弯检测\PolyLaneNet-master\train.py", line 231, in <module>
    model = train(

How should I do to solve it?Thank you :)
